implementations/rust/ockam/ockam_command/src/node/create/config.rs

```diff
@@ -180,25 +182,32 @@ impl NodeConfig {
         self,
         ctx: &Context,
         opts: &CommandGlobalOpts,
+        node_name: &String,
     ) -> miette::Result<()> {
         debug!("Running node config in foreground mode");
         // First, run the `project enroll` commands to prepare the identity and project data
-        self.project_enroll
+        if !self
+            .project_enroll
             .run_in_subprocess(opts.global_args.quiet)?
             .wait()
             .await
-            .into_diagnostic()?;
-
-        let node_name = self.node.name();
+            .into_diagnostic()?
+            .success()
+        {
+            return Err(miette!("Project enroll failed"));
+        }
 
         // Next, run the 'node create' command
-        let mut child = self.node.run_in_subprocess(opts.global_args.quiet)?;
+        let child = self.node.run_in_subprocess(opts.global_args.quiet)?;
+        ctrlc::set_handler(move || {
```

Comment:
an unused `move` here

Reply:
Good catch! I'll remove it in the next iteration to not go over the CI checks again for this, thanks 🙏